src/components/ResearchGroupCard/style.scss

```diff
@@ -1,23 +1,120 @@
-.App {
-  text-align: center;
+$card-header-font: 24px;
+$card-header-height: 64px;
+
+.card{
+    text-align: center;
+    font-family: 'Futura', sans-serif;
+    background-color: #fafafa;
+    border-style: solid;
+    border-color: #cecece;
+    margin: 10px;
+    height: 450px;
+    min-width: 200px;
+    overflow-wrap: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.card_professor{
+    text-align: left;
+    color: #5dadf7;
+    margin-left: 20px;
+    margin-right: 20px;
+    font-size: 16px;
+    height: 5%;
+    margin-top: 20px;
+    overflow-y: hidden;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
+.card_research-group{
+    text-align: left;
+    font-weight: bold;
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 2px;
+    margin-bottom: 15px;
+    font-size: $card-header-font;
+   height: $card-header-height;
+   text-overflow: ellipsis;
+   overflow-y: hidden;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: #61dafb;
+.card_content{
```

Comment:
I see you have a lot of styles for classes that are children of another component. For example, `.card_professor`, `.card_research-group`, `.card_content`, etc. It's good that you preface everything with `.card` to keep in consistent - this semantically indicates that it's a child. 

With regular CSS, you can specifically specify a child of another like so:
```
.card .card_professor {
 /*css here*/
}
.card .card_research-group{
/*css here*/
}
```
You could do this. However, a note I would make is that we're actually using sass, which lets us do fancy shorthand like this:

```
.card {
   /*.card styles go here*/
  .card_professor {
      /*.card_professor styles go here*/
   }
   .card_research-group {
      /*.card_research-group styles go here*/
   }
   /* etc for all other children of .card */
}
```

Please change your CSS to the above style ^^. It let's your styling be more specific, and actually encapsulates styles.

Reply:
Thank you for the detailed explanation! I have now modified it to reflect this style!